README.md

```diff
@@ -84,9 +84,10 @@ This is a *demo* that enables basic, read-only querying of the [Microsoft Graph
 
```

Comment:
I tried testing this but got this error when querying the users collection. It works on the public demo site.
`
{
	users {
    displayName
  }
}
`


```
{
  "errors": [
    {
      "message": "Expected Iterable, but did not find one for field Query.users.",
      "locations": [
        {
          "line": 28,
          "column": 2
        }
      ],
      "path": [
        "users"
      ]
    }
  ],
  "data": {
    "users": null
  }
}
```

Reply:
I believe this is linked to the url/path issue you also documented. After making that change, I was able to make the query here.

Reply:
Upon further investigation, this is due to a permissions issue with some tenants.